neural_modelling/src/neuron/input_types/input_type.h

```diff
@@ -10,17 +10,18 @@ typedef struct input_type_t* input_type_pointer_t;
 //! \param[in] value The value of the input before scaling
 //! \param[in] input_type The input type pointer to the parameters
 //! \return The value of the input after scaling
-static input_t input_type_get_input_value(
-    input_t value, input_type_pointer_t input_type);
+static input_t* input_type_get_input_value(
+    input_t* value, input_type_pointer_t input_type,
+	uint16_t num_receptors);
 
 //! \brief Converts an excitatory input into an excitatory current
 //! \param[in] exc_input The total excitatory input this timestep - note that
 //!     this will already have been scaled by input_type_get_input_value
 //! \param[in] input_type The input type pointer to the parameters
 //! \param[in] membrane_voltage The membrane voltage to use for the input
 //! \return The excitatory input current
-static input_t input_type_convert_excitatory_input_to_current(
-    input_t exc_input, input_type_pointer_t input_type,
+static void input_type_convert_excitatory_input_to_current(
+    input_t* exc_input, input_type_pointer_t input_type,
     state_t membrane_voltage);
 
 //! \brief Converts an inhibitory input into an inhibitory current
```

Comment:
As above re-comment (on line below, but github won't let me do that!)

Reply:
Done